src/resources/js/Pages/Dashboard.vue

```diff
@@ -1,26 +1,33 @@
 <template>
-    <app-layout>
-        <template #header>
-            <h2 class="font-semibold text-xl text-gray-800 leading-tight">
-                Dashboard
-            </h2>
-        </template>
-
-        <div class="py-12">
-            <div class="max-w-7xl mx-auto sm:px-6 lg:px-8">
-                <div class="bg-white overflow-hidden shadow-xl sm:rounded-lg">
-                    <welcome />
-                </div>
-            </div>
-        </div>
-    </app-layout>
+  <u>Vaccinate Ohio Database Statistics:</u><br>
```

Comment:
Nice work implementing this in Vue. We can pretty this up with Tailwind CSS later. If you haven't checked out Tailwind, you might find it interesting. It's pretty nice for a developer who doesn't want to worry too much about styling.